README.md

```diff
@@ -18,6 +18,11 @@ complex molecular systems for molecular dynamics simulations.
 
 To learn more, get started or contribute, check out our [website](http://mbuild.mosdef.org).
 
+### mBuild within the MoSDeF Ecosystem
+<p align="center">
+  <img src="docs/images/mosdef.svg?raw=true" alt="mBuild within the MoSDeF Ecosystem" width="1000" height="1000"/>
```

Comment:
Lets change this to about 50% of its current size. IMO, it seems a bit too large. dropping it to 500x500 might be a nice change.